EIPS/eip-867.md

```diff
@@ -0,0 +1,194 @@
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to the this meta-EIP.  Community members in general are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
+
+The ERP author may indicate in the ERP comments when he/she believes there is sufficient support and request a review by an EIP editor (see EIP-1 for details).  The editor(s) will mark the ERP as “Accepted” or “Rejected”.  As with any EIP, a Rejected ERP may be re-submitted upon substantial improvement.  
+
+Once an ERP is marked as “Accepted” by an EIP Editor, the block at which the changes will be applied will be finalized based on coordination between the client teams and other pending releases.  Once the block is determined, a finalized version (updated to include the determined block) of the State Change Object will be published to the ERP.  Clients will bundle this final version (unmodified) with their respective applications.  
+
+Before the corresponding client updates are propagated on the mainnet, they should first be tested end-to-end on testnet networks, providing a final chance for the ERP stakeholders to monitor a test version of their recovery. Assuming the recovery action is included in an already planned network update, updated clients will already be tested on the testnet before a coordinated release on the mainnet. 
+
+## Ethereum Client Implementation 
+Upon approval of this EIP, clients will implement a module that scans a designated directory for SCO files (each time the client process launches) to construct a mapping between target blocks and SCO file names.  When starting work on a new block, clients should first consult the set of SCO target blocks discovered and determine if there are any recovery actions required for the current block.  
+
+e.g.  in this example `erpsByTargetBlock` is mapping between the target block number associated with each ERP's State Change Object and a reference (i.e. file name) to the resource with the actual data.
```

Comment:
- E.g. _,_ in this
- example _,_ `erpsByTargetBlock`
- is _the_ mapping
- (i.e. _,_ file